python_modules/dagster/dagster/core/definitions/decorators/config_mapping.py

```diff
@@ -0,0 +1,76 @@
+from typing import Any, Callable, Optional, Union
+
+from dagster import check
+from ..config import ConfigMapping
+
+def config_mapping(
+    config_fn: Callable[..., Any] = None,
+    config_schema: Optional[Any] = None,
+    receive_processed_config_values: Optional[bool] = None,
+) -> Union[_ConfigMapping, ConfigMapping]:
+    """Create a config mapping with the specified parameters from the decorated function.
+
+    This shortcut simplifies the core :class:`ConfigMapping` API by omitting additional
+    parameters when they are not needed. The config schema will be inferred from the type
+    signature of the decorated function if not explicitly provided.
```

Comment:
Maybe get rid of the first sentence here, I think it's a bit confusing from the user's perspective.